app/components/progress_label_component.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class ProgressLabelComponent < ViewComponent::Base
```

Comment:
Just wondering if these components can be tested in isolation.

Reply:
Actually, yes they can. Good shout!